src/user/contributing.rst

```diff
@@ -48,5 +48,16 @@ We are glad to know that you would like to contribute to the ``conda-forge`` doc
    your changes.
 5. Submit a `pull request
    <https://help.github.com/articles/about-pull-requests/>`_ to the main repository proposing your changes.
+   
+  
+.. _Guidelines:
+
+Guidelines
```

Comment:
We should probably change "Guidelines" to "Writing Guidelines" as we want to be specific that these are writing guidelines.